src/api/Users.js

```diff
@@ -36,6 +36,19 @@ class Users extends Base {
         return `${this.getBaseApiUrl()}/users/${id}/avatar`;
     }
 
+    /**
+     * API URL for fetching all users in the current user's enterprise
+     *
+     * @param {string} filterTerm Filter for enterprise users
+     * @return {string} URL for fetching enterprise users
+     */
+    getUsersInEnterpriseUrl(filterTerm: string): string {
+        if (!filterTerm) {
+            throw new Error('Missing filter term');
+        }
+        return `${this.getBaseApiUrl()}/users?filter_term=${filterTerm}`;
```

Comment:
Perhaps we need go get in the habit of using [`query-string`](https://www.npmjs.com/package/query-string) to handle these cases. this will automatically handled escaping of `filterTerm` where needed (and if desired).

We already have this package as a dependency.